mezzanine/project_template/project_name/settings.py

```diff
@@ -185,11 +185,11 @@
 # URL that handles the media served from MEDIA_ROOT. Make sure to use a
 # trailing slash.
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
-MEDIA_URL = STATIC_URL + "media/"
+MEDIA_URL = "/media/"
```

Comment:
Just some background info, starting in Django 2.2 there is now a check that runs when a site is in DEBUG mode and will throw an "ImproperlyConfigured" error if the `MEDIA_URL` is inside the `STATIC_URL`.  (It will not raise the error if `DEBUG` is "False".)  From the discussions in the relevant issues ([#29570](https://code.djangoproject.com/ticket/29570) and the earlier [#15199](https://code.djangoproject.com/ticket/15199)) it seems that the Django project strongly wishes to discourage this type of configuration.